Mathlib/Data/Fintype/Basic.lean

```diff
@@ -936,6 +936,9 @@ theorem Fintype.coe_image_univ [Fintype α] [DecidableEq β] {f : α → β} :
 instance List.Subtype.fintype [DecidableEq α] (l : List α) : Fintype { x // x ∈ l } :=
   Fintype.ofList l.attach l.mem_attach
 
+instance List.setOf_fintype [DecidableEq α] (l : List α) : Fintype { x | x ∈ l } :=
```

Comment:
Do you actually need this? If you do, it might be better as an instance `Fintype {x // p x} → Fintype {x | p x}`?

Reply:
It's necessary for the `toFinite_toFinset` call in `setOf_mem_list_eq_singleton_of_nodup`. I changed it to use an implicit instance.